pinot-core/src/main/java/org/apache/pinot/core/operator/query/SelectionOrderByOperator.java

```diff
@@ -117,6 +119,31 @@ public String toExplainString() {
     return stringBuilder.append(')').toString();
   }
 
+  private void pruneOrderByColumns() {
```

Comment:
We don't need to track `_numOrderByColsPreSorted` (the method name does not reflect the functionality as well). Suggest changing this method to `private boolean isAllOrderByColumnsSorted()` and call it in `getNextBlock()`

Reply:
Done.